lib/PreviewProvider.js

```diff
@@ -82,7 +82,7 @@ PreviewProvider.prototype = {
       // filter out the allowed query params and update the query string
       newURL.search = Array.from(queryParams.keys())
         .filter(param => ALLOWED_QUERY_PARAMS.has(param))
-        .map(param => param = param + "=" + queryParams.get(param)).join("&");
+        .map(param => param + "=" + queryParams.get(param)).join("&");
```

Comment:
Yup. Another coding error although not incorrect but dangerous. Reassigning the parameter does nothing.